spark-connector/spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/TestSparkTransformConverter.java

```diff
@@ -42,20 +42,30 @@ void init() {
 
   @Test
   void testPartition() {
```

Comment:
IMO, we could use @ParameterizedTest to generate `supportsBucketPartition` to cover more cases for this test.

Reply:
updated.